frontend/src/components/Catalogue.jsx

```diff
@@ -24,8 +24,8 @@ const Catalogue = () => {
 	const [catalogueDescription, setCatalogueDescription] = useState("");
 
 	const [searchValue, setSearchValue] = useState("");
-	const [sortValue, setSortValue] = useState("newest"); // does nothing atm
-	const [filterValue, setFilterValue] = useState(""); // does nothing atm
+	const [sortValue, setSortValue] = useState("newest");
+	const [filterValue, setFilterValue] = useState(""); // @TODO remove filter?
```

Comment:
Might be best as we don't have a lot of time left

Reply:
Commented out for now.